.github/workflows/auto-publish.yml

```diff
@@ -1,20 +1,22 @@
+# .github/workflows/auto-publish.yml
 name: CI
 on:
   pull_request: {}
   push:
-    branches: [main]
+    branches: [gh-pages]
```

Comment:
Thanks for the PR! But, I'm a bit confused about what's the intended effect of this PR. Now it only does something when pushing to `gh-pages`, instead of when pushing to `main`. That means that it won't auto-publish an editor's draft anymore when merging PRs, for example, right? I assume that's not intended, or is it?

Reply:
Did you mean to push and forgot? If not, you need to clarify what you meant.